modules/test/helpers/service.go

```diff
@@ -22,7 +22,7 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-// CreateService -
+// CreateService - never used(?)
```

Comment:
I think it might be used by https://github.com/openstack-k8s-operators/infra-operator/pull/39